plugin/src/main/java/io/snyk/languageserver/protocolextension/ProgressManager.java

```diff
@@ -12,15 +12,15 @@
 import org.eclipse.lsp4j.jsonrpc.messages.Either;
 
 public class ProgressManager {
-	SnykExtendedLanguageClient lc = SnykExtendedLanguageClient.getInstance();
+	SnykExtendedLanguageClient lc;
```

Comment:
Why is this correct? This could leave the ProgressManager with a null instance of lc, doesn't it?